authenticator/GoogleAuthenticator.cfc

```diff
@@ -72,6 +72,11 @@ component output="false" {
         return 'otpauth://totp/#arguments.email#?secret=#arguments.key#';
     }
 
+    public string function getOTPQRURL(required string OTPURL){
+      local.qrURL = "https://chart.googleapis.com/chart?chs=200x200&cht=qr&chl=200x200&chld=M|0&cht=qr&chl=";
```

Comment:
It seems like a bad idea to send your secret tokens to Google.  They seem nice but I assume this grants thousands of people the ability to grep your referral domain from their logs and get a very small list of possible secrets for any TOTP enabled account on your domain.